packages/block-library/src/site-logo/editor.scss

```diff
@@ -10,20 +10,6 @@
 		pointer-events: none;
 	}
 
-	&:not(.is-default-size) {
```

Comment:
Why did you remove these styles? Is it because this change `const currentWidth = width || defaultWidth;` ? You have kept the addition of this class in code though - should we remove the code as well?

Reply:
Yeah that's correct. However `.is-default-size` is used on the placeholder for when there is no set Site Logo, or the user resets the logo. This class is added to `blockProps` which wraps both the placeholder and the actual Site logo when set, so I just opted to leave it. On the frontend, this class is added if no custom size is set.